lib/plugins/audit.js

```diff
@@ -198,67 +201,80 @@ function auditLogger(opts) {
     }
     var errSerializer = bunyan.stdSerializers.err;
 
+    // don't break legacy use, where there was no top level opts.serializer
     if (opts.log.serializers && opts.log.serializers.err) {
         errSerializer = opts.log.serializers.err;
     }
 
-    var log = opts.log.child({
-        audit: true,
-        component: opts.event,
-        serializers: {
-            err: errSerializer,
-            req: function auditRequestSerializer(req) {
-                if (!req) {
-                    return false;
-                }
+    if (opts.serializers && opts.serializers.err) {
+        errSerializer = opts.serializers.err;
+    }
 
-                var timers = {};
-                (req.timers || []).forEach(function forEach(time) {
-                    var t = time.time;
-                    var _t = Math.floor(1000000 * t[0] + t[1] / 1000);
-                    timers[time.name] = (timers[time.name] || 0) + _t;
-                });
-                return {
-                    // account for native and queryParser plugin usage
-                    query:
-                        typeof req.query === 'function'
-                            ? req.query()
-                            : req.query,
-                    method: req.method,
-                    url: req.url,
-                    headers: req.headers,
-                    httpVersion: req.httpVersion,
-                    trailers: req.trailers,
-                    version: req.version(),
-                    body: opts.body === true ? req.body : undefined,
-                    timers: timers,
-                    connectionState:
-                        req.connectionState && req.connectionState()
-                };
-            },
-            res: function auditResponseSerializer(res) {
-                if (!res) {
-                    return false;
-                }
+    var reqSerializer = function auditRequestSerializer(req) {
```

Comment:
Now that you've broken out the default serializers, it looks like these could probably be lifted to the top as constants/defaults (or maybe even a `DEFAULT_SERIALIZERS` obj). Is that possible, or do we need the closure still?

Reply:
I think we still need the closure because of the " body: opts.body === true ? req.body : undefined,".  We could do something like create a function that returns the DEFAULT_SERIALIZERS and pass it the opts/opts.body param.  It might just be enough to define a DEFAULTS_SERIALIZERS within the clousure, though.  What do you think is more readable?